guides/image_classification_with_vision_transformers.md

```diff
@@ -0,0 +1,55 @@
+# Image Classification with Vision Transformers
+
+related_spaces: abidlabs/vision-transformer
+tags: VISION, TRANSFORMERS, HUB
+
+## Introduction
+
+Image classification is a central task in computer vision. Building better classifiers to classify what object is present in a picture is an active area of research, as it has applications stretching from facial recognition to manufacturing quality control. 
+
+State-of-the-art image classifiers are based on the *transformers* architectures, originally popularized for NLP tasks. Such architectures are typically called vision transformers (ViT). Such models are perfect to use with Gradio's *image* input component, so in this tutorial we will build a web demo to classify images using Gradio. We will be able to build the whole web application in a **single line of Python**, and will look like this (try one of the examples!):
+
+<iframe src="https://hf.space/gradioiframe/abidlabs/vision-transformer/+" frameBorder="0" height="450" title="Gradio app" class="container p-0 flex-grow space-iframe" allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"></iframe>
+
+
+Let's get started!
+
+### Prerequisites
+
+Make sure you have the `gradio` Python package already [installed](/getting_started).
+
+## Step 1 — Choosing a Vision Image Classification Model
+
+First, you will need a image classification model. For this tutorial, we will use a model from the [Hugging Face Model Hub](https://huggingface.co/models). The Hub contains thousands of models covering dozens of different machine learning tasks. 
+
+Expand the Tasks category on the left sidebar and select "Image Classification" as our task of interest. You will then see all of the models on the Hub that are designed to classify images.
```

Comment:
you can also link directly to the models with image classification selected, if you want: https://huggingface.co/models?pipeline_tag=image-classification